angr/analyses/cfg/indirect_jump_resolvers/jumptable.py

```diff
@@ -741,11 +770,13 @@ def _resolve(self, cfg, addr, func_addr, b):
             return False, None
         if jump_target is not None:
             if self._is_target_valid(cfg, jump_target):
-                ij = cfg.indirect_jumps[addr]
-                ij.jumptable = False
-                ij.resolved_targets = { jump_target }
+                ij = cfg.indirect_jumps.get(addr, None)
```

Comment:
I want to refactor this cfg.indirect_jumps access out of here, but decided to do it for a follow up